api/Document.json

```diff
@@ -5959,7 +6131,7 @@
               {
                 "version_added": "10",
                 "version_removed": "65",
-                "alternative_name": "onmozfullscreenchange"
+                "alternative_name": "mozfullscreenchange"
```

Comment:
It looks this PR removes the `on-` prefix from alternative names even for the `on-` event handler property, which I don't think is correct.